src/elm/Page/Dashboard/TransferScreen.elm

```diff
@@ -0,0 +1,374 @@
+module Page.Dashboard.TransferScreen exposing (Model, Msg, init, msgToString, subscriptions, update, view)
```

Comment:
we should avoid the wording "Screen" Altough we have lots of them, none of the pages is actually called that way. We will have a transfer form, and a transfer view. Maybe it makes more sense to have a folder to group transfer stuff

Reply:
Also, this is under "/Dashboard" even tough it doesn't make part of the dashboard screens

Reply:
I concluded it is under "Dashboard" because in the new design https://github.com/cambiatus/frontend/issues/41#issuecomment-561199062 the Dashboard bar is highlighted.

Reply:
I see, ok!